---
title: Blok 'Catch' nigdy nie został osiągnięty, ponieważ '<name1>'dziedziczy po'<name2>'
ms.date: 07/20/2015
f1_keywords:
- BC42029
- vbc42029
helpviewer_keywords:
- BC42029
ms.assetid: 4bc84a84-ec7e-49db-8794-4fb71d36d5c0
ms.openlocfilehash: 59dfa9cbc80bc9cc3faf07923d0d579070750426
ms.sourcegitcommit: 9b552addadfb57fab0b9e7852ed4f1f1b8a42f8e
ms.translationtype: HT
ms.contentlocale: pl-PL
ms.lasthandoff: 04/23/2019
ms.locfileid: "62003694"
---
# <a name="catch-block-never-reached-because-name1-inherits-from-name2"></a>Blok 'Catch' nigdy nie został osiągnięty, ponieważ '\<name1 >' dziedziczy po elemencie '\<Nazwa2 >'
A `Catch` bloku w kodzie nigdy nie zostanie osiągnięty.  
  
 Domyślnie ta wiadomość jest ostrzeżenie. Aby uzyskać więcej informacji na temat ukrywania ostrzeżenia lub traktowanie ostrzeżeń jako błędy, zobacz [Konfigurowanie ostrzeżeń w języku Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).  
  
 **Identyfikator błędu:** BC42029  
  
## <a name="to-correct-this-error"></a>Aby poprawić ten błąd  
  
- Usunąć obce `Catch` bloku.  
  
## <a name="see-also"></a>Zobacz także

- [Jak przechwytywać wyjątki za pomocą bloku try/catch](../../standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md)
- [Try...Catch...Finally, instrukcja](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
